kinisi/analyzer.py

```diff
@@ -61,29 +63,60 @@ def _from_Xdatcar(cls,
         :param progress: Print progress bars to screen. Optional, defaults to :py:attr:`True`.
         """
         if dtype is None:
-            p = PymatgenParser(trajectory.structures, specie, time_step, step_skip, dt, dimension, distance_unit, progress)
+            p = PymatgenParser(trajectory.structures, specie, time_step, step_skip, dt, dimension, distance_unit,
+                               progress)
             return cls(p)
         elif dtype == 'identical':
-            u = [PymatgenParser(f.structures, specie, time_step, step_skip, dt, dimension, distance_unit, progress) for f in trajectory]
+            u = [
+                PymatgenParser(f.structures, specie, time_step, step_skip, dt, dimension, distance_unit, progress)
+                for f in trajectory
+            ]
             p = u[0]
             p.displacements = sc.concat([i.displacements for i in u], 'atom')
             return cls(p)
         elif dtype == 'consecutive':
             structures = _flatten_list([x.structures for x in trajectory])
             p = PymatgenParser(structures, specie, time_step, step_skip, dt, dimension, distance_unit, progress)
             return cls(p)
-        
+
+    @classmethod
+    def _from_Universe(cls,
```

Comment:
```suggestion
    def _from_universe(cls,
```

Python methods should be snake_case.